doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -135,6 +135,9 @@ To enable one of the reactions to the last fabric removal, set the corresponding
   When the :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option is also set to ``y``, the device will also remove all non-volatile data stored on the device, including application-specific entries.
   This means the device is restored to the factory settings.
 
+To create a delay between  the chosen reaction and the last fabric being removed, set the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_SCHEDULE_TIME` Kconfig option to a specific time in milliseconds.
```

Comment:
You didn't modify the Kconfig name accordingly to the recent changes in corresponding PR.

Reply:
Fixed.